### DISCUSSION TITLE
Pro version, also copy color name

### DISCUSSION THREAD

**Discussion by Zillion01** [rsayfutdinov/Pikka#10] [Ideas]
Feature request :)

I just bought the pro version. Very good!

Still buggy,  color picker sometimes only  shows one color when picking. After restart of app it works again. Mac 11.4, Pikka 2.3.0 (? how can I see the version number).

 I would like:

- easy copy the CSS hex code with one click, as already implemented
- aditional  I would like to be able to also easily copy the color name in camelCase and dashed-name.

**Comment by rsayfutdinov** [comment]
> Feature request :)
> 
> 
> 
> I just bought the pro version. Very good!
> 
> 
> 
> Still buggy,  color picker sometimes only  shows one color when picking. After restart of app it works again. Mac 11.4, Pikka 2.3.0 (? how can I see the version number).
> 
> 
> 
>  I would like:
> 
> 
> 
> - easy copy the CSS hex code with one click, as already implemented
> 
> - aditional  I would like to be able to also easily copy the color name in camelCase and dashed-name.
> 
> 

Hi,
Thank you for the feature request.
What did you mean by "only shows one color when picking"? To pick color sequentially you need to hold Shift key on the keyboard.

> - easy copy the CSS hex code with one click, as already implemented

Do you mean to copy hex values from the library with a single click? 

> - aditional  I would like to be able to also easily copy the color name in camelCase and dashed-name.

I'm thinking about adding custom color formats, so one can define any format which would fit the needs.

Regards,
Ruslan

**Reply by Zillion01** [reply]
Thanks for your comment. The bug was the multi display bug, seems to be fixed with 2.3.2.

I like this tool because I am color blind and webdeveloper. So I like that it says the name of the color, like "Light Grey", "Greenish Grey" etc. I use them also in my CSS / SASS code, so would be cool if I could immediately copy "$lightGrey: #dcdcdc;" I could put this directly in my code. Or allow to make a format to copy including the colorname and the color format.

**Reply by rsayfutdinov** [reply]
Great to hear that you like Pikka. I really appreciate your feedback.
The feature with custom color formats is already on my list. ( https://github.com/rsayfutdinov/Pikka/projects/1 )

PS: May I ask you to drop a review on mac app store it would be a huge help for me and will support further Pikka development ?

**Reply by Zillion01** [reply]
Great, looking forward to it. Will write a review!

Another idea btw, I guess the tool will be used a lot by app and web developers.. to add as a bonus (or pro feauture) a ruler, so you can measure stuf on the screen in pixels.

**Reply by rsayfutdinov** [reply]
> Great, looking forward to it. Will write a review!
> 
> Another idea btw, I guess the tool will be used a lot by app and web developers.. to add as a bonus (or pro feauture) a ruler, so you can measure stuf on the screen in pixels.

I will think about the screen ruler. This is definitely a good idea, but not so easy to implement.
